### ISSUE TITLE
Any advice on adding a CNN backbone？

### ISSUE THREAD

**Issue by Taeboush** [microsoft/Swin-Transformer#61]
i want to add a CNN like ResNet50 or VGG as backbone, which output feature map as the input of swin-transformer
cause my own dataset only include a few thousands images, so i don't want to take the raw image or just normalized image as the input of swin-transformer
Any advice on adding a CNN backbone, requirement of the input size of swin-transformer seems to be strict.

**Comment by zeliu98** [comment]
Hi, for custom input size you can simply add some paddings,  our detection and segmentation repo give some example:
- https://github.com/SwinTransformer/Swin-Transformer-Object-Detection/blob/6a979e2164e3fb0de0ca2546545013a4d71b2f7d/mmdet/models/backbones/swin_transformer.py#L433-L436 
- https://github.com/SwinTransformer/Swin-Transformer-Object-Detection/blob/6a979e2164e3fb0de0ca2546545013a4d71b2f7d/mmdet/models/backbones/swin_transformer.py#L284-L286
- https://github.com/SwinTransformer/Swin-Transformer-Object-Detection/blob/6a979e2164e3fb0de0ca2546545013a4d71b2f7d/mmdet/models/backbones/swin_transformer.py#L215-L218
- ...

**Comment by ancientmooner** [comment]
When you have only a few thousands images, one way to leverage Swin Transformer is to do fine-tuning from the pre-trained models.

**Comment by wutongshu819274135** [comment]
> 我想添加一个像 ResNet50 或 VGG 这样的 CNN 作为主干，它输出特征图作为 swin-transformer 的输入， 因为我自己的数据集只包含几千张图像，所以我不想拍摄原始图像或只是标准化图像作为 swin-transformer 的输入 任何关于添加 CNN 主干的建议，对swin-transformer的输入大小的要求似乎很严格。

你好，我也是小数据集可以交流一下吗qq819274135

**Comment by wutongshu819274135** [comment]
> i want to add a CNN like ResNet50 or VGG as backbone, which output feature map as the input of swin-transformer cause my own dataset only include a few thousands images, so i don't want to take the raw image or just normalized image as the input of swin-transformer Any advice on adding a CNN backbone, requirement of the input size of swin-transformer seems to be strict.

hello，I also use a small data set. Can I exchange it

**Comment by wutongshu819274135** [comment]
> i want to add a CNN like ResNet50 or VGG as backbone, which output feature map as the input of swin-transformer cause my own dataset only include a few thousands images, so i don't want to take the raw image or just normalized image as the input of swin-transformer Any advice on adding a CNN backbone, requirement of the input size of swin-transformer seems to be strict.

hello，I also use a small data set. Can I exchange it。qq：819274135
